WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -1,5 +1,12 @@
 import UIKit
 
+// NSNotification sent when a site or a tag is unfollowed via Reader Manage screen.
```

Comment:
FWIW, I did initially try to use delegates. But it got messy due to all the classes that are involved just to display the tags and sites view controllers. This ended up being much cleaner/less convoluted.

Reply:
I really like this approach :)  Totes agree delegates be more complicated.